dataikuapi/fmclient.py

```diff
@@ -210,6 +213,27 @@ def list_instance_images(self):
         """
         return self._perform_tenant_json("GET", "/images")
 
+    def get_instance_client(self, instance_id, api_key):
+        """
+        Get a Python client to communicate with an instance
+
+        :param str instance_id: The instance ID
+        :param str api_key: The API key generated from the target DSS instance
+
+        :return: a Python client to communicate with the target instance
+        :rtype: :class:`dataikuapi.dssclient.DSSClient`
+        """
+        instance = self.get_instance(instance_id)
+        instance_status = instance.get_status()
+        public_url = instance_status.get("publicURL")
+
+        # TODO waiting for PR merge to be uncommented
+        # https://github.com/dataiku/dataiku-api-client-python/pull/245
+        # if instance.instance_data.get("nodeType") == "govern":
+            # return GovernClient(public_url, api_key)
```

Comment:
a warning message will be welcome, we may forget the comment ...

Reply:
I prefer keeping the comment as the PR is already ongoing
we will probably have to coordinate with them:
* if we merge first, they uncomment the code in their PR
* if they merge first, we remove the exception raising and implement directly the `GovernClient` correctly